src/checkstyle/checkstyle.xml

```diff
@@ -178,4 +178,4 @@
 		<module name="WhitespaceAround" />
 
 	</module>
-</module>
+</module>
```

Comment:
We are coming back to this issue.
Each file has to have a new line in the end.
The IDE has a particular setting on the matter.

Although it is fully unclear what do you do in these files if we don't see any changes, only this noise :smile: